src/pages/Blogs.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import Bloglist from '../components/blogs/Bloglist';
+
+import Container from '../components/UI/Container';
+
+const Blogs = () => {
+  return (
+    <Container>
+      <section className="min-h-screen">
+        <Bloglist />
```

Comment:
Blogs page needs to be redesigned and to add a swiper 
single blog the sharing icons should be on the image's right-hand side not below